pkg/evpn/evpn.go

```diff
@@ -90,6 +90,21 @@ func (n *NLRI) GetEVPNLabel() []uint32 {
 	return label
 }
 
+// GetEVPNRAWLabel returns stack of raw labels.
```

Comment:
It looks to me that it is not 100% EVPN specific function. Other protocols might be also interested in getting the raw label. Please extend `base.Label`  with new method Raw which would return uint32 of the raw label value. In this case computing of raw value will be in a single location and it does not need to be implemented by every NLRI which needs it,

Reply:
From programing point of view best thing is chaning label structure to inclue only raw label and assign methods to get exp and bos bit like .GetExp and GetBos. But I gues this will require more change and and cause trouble where they are used. Although I do not see the use of exp and bos bit in the code. Nevertheless it will requrire lots of change on test methods:)) So I will add GetRawValue which will be a little bit ugly:)

Reply:
Ok figured it out. But this need change evpn/evpn.go GetEVPNLabel method to return labels itself.